pkg/investigations/mapping/mapping.go

```diff
@@ -20,6 +21,8 @@ func GetInvestigation(alertTitle string) *investigation.Investigation {
 		return investigation.NewInvestigation(chgm.Investigate, "ClusterHasGoneMissing")
 	case strings.Contains(alertTitle, "ClusterProvisioningDelay -"):
 		return investigation.NewInvestigation(cpd.Investigate, "ClusterProvisioningDelay")
+	case strings.Contains(alertTitle, "InsightsOperatorDownSRE"):
+		return investigation.NewInvestigation(insightsoperatordown.Investigate, "InsightsOperatorDown")
```

Comment:
Accidentally commited this? I don't see this existing in the PR.

Reply:
ah jup ^^ coming in another PR